render.js

```diff
@@ -8,14 +8,23 @@ var contentsCb = function(pobj) {
   });
 }
 
-if (phantom.args.length < 2) {
+var args;
+
+if (phantom.args) {
+    args = phantom.args
+}
+else {
+    args = require('system').args;
```

Comment:
@AlexandreAWE the fix is correct, but please notice that system.args contains the script name as the first argument. So head should be removed from args in this case.
[Docs](http://phantomjs.org/api/phantom/property/args.html)